test/functional/fundrawtransaction.py

```diff
@@ -452,7 +452,7 @@ def run_test(self):
         self.stop_node(2)
         self.stop_node(3)
         self.nodes[1].encryptwallet("test")
-        bitcoind_processes[1].wait(timeout=BITCOIND_PROC_WAIT_TIMEOUT)
+        self.bitcoind_processes[1].wait(60)
```

Comment:
Please keep the arg named. I'd really prefer if all primitive types are named in our python code.